src/main/java/io/github/thebusybiscuit/sensibletoolbox/listeners/PlayerUUIDTracker.java

```diff
@@ -34,10 +34,7 @@ public void loadMap() {
                 map.load(file);
                 saveNeeded = false;
             }
-            catch (IOException e) {
-                e.printStackTrace();
-            }
-            catch (InvalidConfigurationException e) {
+            catch (IOException e|InvalidConfigurationException e) {
```

Comment:
Both variables are named `e`, that's why the build is failing. You'll have to rename one of these